cmd/util/cmd/read-badger/cmd/commits.go

```diff
@@ -18,8 +17,6 @@ func init() {
 
 	commitsCmd.Flags().StringVarP(&flagBlockID, "block-id", "b", "", "the block id of which to query the state commitment")
 	_ = commitsCmd.MarkFlagRequired("block-id")
-
-	rootCmd.AddCommand(findBlockByCommits.Init(InitStorages))
```

Comment:
the `findBlockByCommit` should be a subcommand of blocks not commits. Removing for now, we can re-implement again later.